packages/cli/src/utils/index.js

```diff
@@ -158,3 +159,7 @@ ${chalk.bold('DeprecationWarning: Starting with Nuxt version 3 this will be a fa
     exit(0)
   }
 }
+
+export function createLock(...args) {
```

Comment:
This might seem redundant but I had a problem with mocking this function when I immeditately exported it. Jest threw: `TypeError: Cannot set property createLock of #<Object> which has only a getter`

Reply:
Please add a comment above explaining about the reason.